src/camvidlog2/yoloe/cli.py

```
@@ -0,0 +1,65 @@
+        if not video.exists():
+            print("File does not exist")
+            print(video)
+            continue
+
+        # inference
+        array = pd.concat(
+            generate_tracked_bboxes(
+                (v for _, v in generate_frames_cv2(video)),
+                onnx_path=onnx,
+                class_names=classes,
+            )
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)